infrastructure/deploy/stream_authorizer.tf

```diff
@@ -53,35 +53,43 @@ resource "aws_iam_role_policy_attachment" "stream_authorizer_basic_execution_rol
 }
 
 locals {
-  source_sha = sha1(join("", [for f in fileset(path.module, "../../lambdas/stream-authorizer/*.{js,json}") : sha1(file(f))]))
+  auth_source_path        = "${path.module}/../../lambdas/stream-authorizer"
+  auth_allowed_referers   = "(devbox|digitalcollections|meadow|dc)\\.(rdc(-staging)?\\.)?library\\.northwestern\\.edu"
```

Comment:
Do we want to change this so it doesn't match everything with `.rdc`  like `foo.rdc.library.northwestern.edu` ?

Reply:
Yes. I'll update it to use a variable and then update the staging and production tfvars files with a more closely tailored regex for each environment.

- Staging: `/\b(devbox|digitalcollections|meadow|dc)\.rdc-staging\.library\.northwestern\.edu/`
- Production: `/\b(digitalcollections|meadow|dc)(\.rdc)?\.library\.northwestern\.edu/`

The `\b` will make sure that the first match starts on a word break (i.e., `https://dc` will match `/\bdc/` but `https://dev.rdc` will not)

Reply:
👍 . I'll let Brendan finish reviewing it. I had just noticed that.